tests/queries/0_stateless/01093_cyclic_defaults_filimonov.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE test
 (
-    `a1` UInt64 DEFAULT a + 1,
```

Comment:
Interesting, how it was possible to create this table? Maybe identical columns just collapsed?

Reply:
It was not possible https://github.com/ClickHouse/ClickHouse/pull/16612/files#diff-bc57ecabc455a3cc24084ccda135da53206df9e520bbe2203c42312f34f1b527R9